src/KafkaFlow.SchemaRegistry/ClusterConfigurationBuilderExtensions.cs

```diff
@@ -1,10 +1,12 @@
-namespace KafkaFlow.Serializer.ApacheAvro
+namespace KafkaFlow.SchemaRegistry
```

Comment:
You can use here the namespace `KafkaFlow` only and set the root namespace of the project as `KafkaFlow` only too